src/systemd_ctypes/bus.py

```diff
@@ -236,7 +236,7 @@ def _coroutine_task_complete(self, out_types: Sequence[str], task: asyncio.Task)
         except BusError as exc:
             self.reply_method_error(exc)
 
-    def reply_method_function_return_value(self, out_types: Sequence[str], return_value: Any) -> bool:  # Literal[True]:
```

Comment:
I wonder why I had this as a comment.  `Literal[True]` is clearly the correct thing to have here, not `bool`, but something was stopping that from working.

I'd prefer not to get rid of this completely without figuring it out first.

Reply:
OK, I put it back and split the arguments to separate lines instead.